address/book.go

```diff
@@ -443,3 +473,7 @@ func (b *Book) RemoveSubscriber(
 
 	return nil
 }
+
+func (b *Book) ProofCourierAddr() ProofCourierAddr {
```

Comment:
nit: missing Godoc. And do we really need this?

Reply:
This is only used in a unit test where we need a new random tap address to have the same proof courier address as that found in the address book. Here:
https://github.com/ffranr/taproot-assets/blob/add-proof-courier-to-addr/tapgarden/custodian_test.go#L168-L171

Reply:
You can just use a random address in that code. Again, this should go away once we persist the proof courier address. Then the address book doesn't need a source of the default courier as that will all be handled on the RPC level (either the user specifies a specific courier URI to use for an address they create or we fall back to the default one configured in the daemon).